wmt-worker/app/services/data/get-pending-tasks-and-mark-in-progress.js

```diff
@@ -0,0 +1,35 @@
+const config = require('../../../knexfile').development
+const knex = require('knex')(config)
+const taskStatus = require('../../constants/task-status')
+const Task = require('../domain/task')
+
+module.exports = function (schema, batchSize) {
+  return knex.select().table('tasks')
+    .where('status', taskStatus.PENDING)
+    .orderBy('date_created', 'asc')
+    .limit(batchSize)
+    .then(function (results) {
+      var tasks = []
+      var ids = []
+      if (results) {
```

Comment:
If there are no tasks returned this will be an empty array, which will be truthy. Therefore this should probably be testing array length instead.